test/stdlib/runtime_autorelease_optimization.txt

```diff
@@ -0,0 +1,101 @@
+// UNSUPPORTED: linux
```

Comment:
Better to make this `REQUIRES: objc_interop`. This won't work on Windows or Android either, after all.

Reply:
Good point